index.bs

```diff
@@ -471,7 +471,7 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
             </li>
             <li>
               <dfn for='HdrMetadataType' enum-value>smpteSt2094-40</dfn>,
-              representing the dynamic metadata type defined by 
```

Comment:
Thanks for fixing this. May I ask whether there is a tool like `git cl format` for W3C spec edits?

Reply:
Not that I'm aware of. I just caught this by accident when cleaning up my own trailing spaces.